src/main/resources/api/api.yaml

```diff
@@ -250,6 +404,79 @@ definitions:
   #
   # Questions
   #
+  V1Question:
+    type: object
+    required:
+      - text
+      - type
+      - topic
```

Comment:
may be it is a good idea to make answers to be required, how do you think?

Reply:
We tell about answers for admin or for testees? If only admin see the answers, they have to be  required, because system must have right answer to any question. So, what see testees: answers or smt else?